09_timer/timer.rb

```diff
@@ -0,0 +1,29 @@
+class Timer
+    def initialize()
+      @seconds = 0
+    end
+    def seconds=(seconds)
```

Comment:
In Ruby we usually use `attr_writer` for setters, `attr_reader` for getters and `attr_accessor` to combine both of them.